### DISCUSSION TITLE
Error when building for production

### DISCUSSION THREAD

**Discussion by Glenn** [pdfme/pdfme#137] [General]
I'm sorry for asking - as this my well be a problem with my understanding of React (I'm a Vue and Angular Dev not a React one - sorry!)

But every time I try to  render anything after building for production I get a babel error thrown saying 

ERROR: An error occurred: *"a" is read-only

I've even tried building the pdfme-playground from scratch and I get the same problem. All I did was:

npm i
npm run build
serve build

on a clean download of the repo

To add extra context to this:

- The error was always in hooks in the init function.
- I tried forking the repo and re-built it without minimizing and it works fine - which is annoying!
- I'm using Node 18 - that could be the issue - I'll try tomorrow with a lower node version just in case

Glenn

**Comment by riicardojs** [comment]
did you get any solution? I'm facing the same problem, after performing the build this error appears. Using Node v16 or v18 the error remains
![image](https://github.com/pdfme/pdfme/assets/32929415/109519b4-fd24-4162-ac5c-1a4aff6ef00d)
![image](https://github.com/pdfme/pdfme/assets/32929415/06f42033-f292-4c9d-857a-e122ee5dd284)

@hand-dot

**Reply by matheusmachadodev1** [reply]
i'm having the same problem, whenever I try to run build

**Reply by Glenn** [reply]
no sorry - I just ended up using a non-minimized build which I pushed as my own npm repo - which is a less than great solution - but I couldn't easily see another fix and that didn't exactly seam like a hack I could contribute back to the community with!
